frontend/components/Card.tsx

```diff
@@ -12,7 +13,25 @@ const StyledCard = styled(AntdCard)`
   flex-direction: column;
   font-size: 0.8rem;
   padding: 1rem;
-  margin: 0.25rem;
+  width: 30%;
+  max-width: 281px;
```

Comment:
can remove this max-width line, as it gets overridden in the media queries

Reply:
<img width="1690" alt="Screen Shot 2022-02-22 at 11 28 38 AM" src="https://user-images.githubusercontent.com/68393684/155175566-752bd737-68f2-4a40-a816-ddaaa1510092.png">

If I was to remove this Stefan, the cards in the navbar mode would grow forever like shown in this picture.